File: 20221019/모범답안/20221019_01/minumum.py
def solution(arr, brr):
    '''
    :param arr: list
    :param brr: list
    :return: int
    '''
    for i in range(len(arr)):
        x, y = min(arr[i], brr[i]), max(arr[i], brr[i])
        arr[i] = x
        brr[i] = y
        
    return max(arr) * max(brr)

File: 20221019/모범답안/20221019_01/minumum.js
/**
 * @param {array} arr
 * @param {array} brr
 * @return {int}
 */
function solution(arr, brr){
    var maxA = 1;
    var maxB = 1;
    for (var j = 0; j < arr.length; j++) {
        if (arr[j] < brr[j]) {
            var temp = arr[j];
            arr[j] = brr[j];
            brr[j] = temp;
        }
        
        maxA = Math.max(maxA, arr[j]);
        maxB = Math.max(maxB, brr[j]);
    }
    
    return maxA * maxB;
}
